### PR TITLE
Update localize.py

### PR DESCRIPTION
**Pull request by ngreis**
Think i messed up previous commit change :S
Now should be ok.

ngreis

### CODE REVIEW

File: custom_components/plex_assistant/localize.py [hunk 1]
```diff
@@ -1016,7 +1016,7 @@
         },
     },
     "pt": {
-        "play": "ver",
+        "play": "play",
```
Comment (maykar) [inline custom_components/plex_assistant/localize.py:1019]: should this be "play" or "ver"?

File: custom_components/plex_assistant/localize.py [hunk 2]
```diff
@@ -1158,36 +1184,97 @@
...
+                "da",
+                "do",
+            ],
         },
         "music_separator": {
-            "keywords": [],
+            "keywords": [
+                "pelo",
+                "pela",
+            ],
```
Comment (maykar) [inline custom_components/plex_assistant/localize.py:1270]: I'm going to need to modify the code to accept multiple keywords for the separator here before I pull this, but it shouldn't be an issue.

File: custom_components/plex_assistant/localize.py (hunk 1 above)
Comment (maykar) [reply custom_components/plex_assistant/localize.py:1019]: It can also be both if necessary:
```
"play": [
    "ver",
    "play",
],
```

**Comment by ngreis** [comment]
Play, can be “ver” in case its movies or tv shows, but for music we say play.

In fact in almost cases nowadays we say play for all, that’s why I changed it.

 

But keep both, for both types of people 😝

 

Best regards

Ngreis

 

De: Ryan Meek <notifications@github.com> 
Enviada: 22 de fevereiro de 2021 14:56
Para: maykar/plex_assistant <plex_assistant@noreply.github.com>
Cc: ngreis <ngreis@sapo.pt>; Author <author@noreply.github.com>
Assunto: Re: [maykar/plex_assistant] Update localize.py (#101)

 

@maykar commented on this pull request.

  _____  

In custom_components/plex_assistant/localize.py <https://github.com/maykar/plex_assistant/pull/101#discussion_r580313083> :

> @@ -1016,7 +1016,7 @@
         },
     },
     "pt": {
-        "play": "ver",
+        "play": "play",

should this be "play" or "ver"?

**Pull request merged**